src/BTVNanoCommissioning/workflows/ctag_Wc_valid_sf.py

```diff
@@ -204,24 +252,24 @@ def process_shift(self, events, shift_name):
         jetindx = jetindx[:, 0]
 
         # Other cuts
-        req_pTratio = (soft_muon[:, 0].pt / mu_jet[:, 0].pt) < 0.4
-        idx = np.where(iso_muon.jetIdx == -1, 0, iso_muon.jetIdx)
-        req_QCDveto = (
-            (iso_muon.pfRelIso04_all < 0.05)
-            & (abs(iso_muon.dz) < 0.01)
-            & (abs(iso_muon.dxy) < 0.002)
-            & (iso_muon.sip3d < 2)
-            & (
-                iso_muon.pt
-                / ak.firsts(
-                    events.Jet[
-                        (events.Jet.muonIdx1 == iso_muindx)
-                        | ((events.Jet.muonIdx2 == iso_muindx))
-                    ].pt
-                )
-                > 0.75
-            )
-        )
+        req_pTratio = (soft_muon[:, 0].pt / mu_jet[:, 0].pt) < muonpTratioCut
+        idx = np.where(iso_lep.jetIdx == -1, 0, iso_lep.jetIdx)
+        # req_QCDveto = (
+        #     (iso_lep.pfRelIso04_all < 0.05)
+        # & (abs(iso_lep.dz) < isolepdz)
+        # & (abs(iso_lep.dxy) < isolepdxy)
+        # & (iso_lep.sip3d < isolepsip3d)
+        # & (
+        #     iso_lep.pt
+        #     / ak.firsts(
+        #         events.Jet[
+        #             (events.Jet.muonIdx1 == iso_lepindx)
+        #             | ((events.Jet.muonIdx2 == iso_lepindx))
+        #         ].pt
+        #     )
+        #     > 0.75
+        # )
+        # )
```

Comment:
make the comments of the old selections, refers to the selections of the run2 project? 
https://github.com/mondalspandan/VHcc-cTagSF